internal/handlers/pg/fetch.go

```diff
@@ -73,3 +74,40 @@ func nextRow(rows pgx.Rows) (*types.Document, error) {
 
 	return doc.(*types.Document), nil
 }
+
+func (h *Handler) protoFetch(ctx context.Context, param fetchParam) ([]*types.Document, error) {
+	var sql string
+	if param.comment != "" {
+		param.comment = strings.ReplaceAll(param.comment, "/*", "/ *")
+		param.comment = strings.ReplaceAll(param.comment, "*/", "* /")
+		param.comment = fmt.Sprintf("/* %s */", param.comment)
+
+		sql = fmt.Sprintf(`SELECT %s _jsonb FROM %s`, param.comment, pgx.Identifier{param.db, param.collection}.Sanitize())
+	} else {
+		sql = fmt.Sprintf(`SELECT _jsonb FROM %s`, pgx.Identifier{param.db, param.collection}.Sanitize())
+	}
+
+	rows, err := h.pgPool.Query(ctx, sql)
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+	defer rows.Close()
+
+	var res []*types.Document
+	for {
+		doc, err := nextRow(rows)
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			return nil, lazyerrors.Error(err)
+		}
+		res = append(res, doc)
+	}
+
+	return res, nil
+}
+
+type fetchParam struct {
```

Comment:
Let's add documentation comment and move it before function that uses it

Reply:
Done.